### PR TITLE
Update PPP and remove dead code.

### PR DESCRIPTION
**Pull request by Diwu-Yi**

### CODE REVIEW

**Review by yanbingtao — DISMISSED** [review]
LGTM

**Review by yanbingtao — DISMISSED** [review]
LGTM

File: docs/team/diwu-yi.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  * Took charge of overall testing of functionalities of tCheck.
+  * Monitored test cases written for all teammates and reminded teammates to add tests when the test coverage is not satisfactory.
+
+* **Documentation**:
+
+  * AboutUs page:
+    * Updated my information in the AboutUs page of the project website.
+ 
+  * User Guide:
+    * Added documentation for the features `i-set i/INGREDIENT_NAME m/AMOUNT` , `i-set-default` and `i-set-all M/AMOUNT_FOR_MILK P/AMOUNT_FOR_PEARL B/AMOUNT_FOR_BOBA L/AMOUNT_FOR_BLACK_TEA G/AMOUNT_FOR_GREEN_TEA S/AMOUNT_FOR_SUGAR`.
```
Comment (ureshiiYing) [inline docs/team/diwu-yi.md:52]: I think it is okay to leave out the parameters, but this is just a note.

File: src/main/java/seedu/address/logic/commands/ingredientcommands/IngredientSetAllCommand.java [hunk 2]
```diff
@@ -80,7 +80,39 @@ public CommandResult execute(Model model) throws CommandException {
         List<Ingredient> lastShownList = model.getFilteredIngredientList();
 
         IngredientBook toSet = new IngredientBook();
-        IngredientBook filledBook = executeHelper(toSet);
+        IngredientBook filledBook = IngredientSetCommand.fillIngredientBookHelper(toSet);
+
```
Comment (ureshiiYing) [inline src/main/java/seedu/address/logic/commands/ingredientcommands/IngredientSetAllCommand.java:84]: Extra blank line?

**Review by ureshiiYing — COMMENTED** [review]
Overall LGTM, except for the one extra empty line.

**Review by Persdre — COMMENTED** [review]
LGTM

**Review by Persdre — DISMISSED** [review]
LGTM

File: docs/team/diwu-yi.md (hunk 1 above)
Comment (Diwu-Yi) [reply docs/team/diwu-yi.md:52]: Okay, I will take note of this.

File: src/main/java/seedu/address/logic/commands/ingredientcommands/IngredientSetAllCommand.java (hunk 2 above)
Comment (Diwu-Yi) [reply src/main/java/seedu/address/logic/commands/ingredientcommands/IngredientSetAllCommand.java:84]: true, will remove.

**Review by yanbingtao — DISMISSED** [review]
LGTM

**Comment by Diwu-Yi** [comment]
Resolve conflicts

**Review by yanbingtao — APPROVED** [review]
LGTM

**Review by Persdre — APPROVED** [review]
LGTM

**Pull request merged**